models/_sector/depin/depin_revenue.sql

```diff
@@ -0,0 +1,18 @@
+{{ config(materialized="table") }}
```

Comment:
We may need the schema and other information in the config blocks:
```suggestion
{{ config(
  schema = 'depin'
  , alias = 'revenue'
  , materialized = 'table'
) }}
```

Reply:
ah helpful - was wondering why build was failing!